magic-links/.env

```diff
@@ -0,0 +1,23 @@
+# required: true
+# format: sid
+# description: Your Twilio Account SID. Can be found in the Console.
+# link: https://www.twilio.com/console
+TWILIO_ACCOUNT_SID=
```

Comment:
Can you change this and auth token to be ACCOUNT_SID and AUTH_TOKEN? They are automatically injected by Functions and the Serverless Toolkit

Reply:
I see they are already not being directly used in your code. So you can just drop these two from here

Reply:
gotcha, cool, I wasn't sure if I needed them!